commands/Moderation/mute.js

```diff
@@ -0,0 +1,58 @@
+exports.init = async (client) => {
+  const schema = client.settings.guilds.schema;
+  if (!schema.muteRoleName) {
+    await client.settings.guilds.add("muteRoleName", "MUTEROLENAME");
+  }
+  if (!schema.muteRoleId) {
+    await client.settings.guilds.add("muteRoleId", "MUTEROLEID");
+  }
+  if (!schema.titleURL) {
+    await client.settings.guilds.add("titleURL", "TITLEURL");
+  }
+  if (!schema.reportChannelId) {
+    await client.settings.guilds.add("reportChannelId", "CHANNELIDHERE");
+  }
+};
+
+exports.run = async (client, msg, [user]) => {
+  try {
+    const value = msg.mentions.members.first().roles.get(muteRoleId);
+    if (msg.mentions.members.first().roles.get(muteRoleId)) {
```

Comment:
You would need to get the settings again for use in the command. You can use this trick to get all the variables you need.
```js
const { muteRoleId, reportChannelId, titleURL, muteRoleName } = msg.guild.settings
```